website_event_snippet_calendar/controllers/main.py

```diff
@@ -54,12 +55,23 @@ def events_for_day(self, day=None, limit=None):
         :param limit int:
             How many results to return.
         """
+
+        searches = parse_qsl(searches[1:])
         ref = day or Date.to_string(date.today())
         domain = [
             ("date_end", ">=", ref),
         ]
         if day:
             domain.append(("date_begin", "<=", ref))
+
+        for search in searches:
+            if search[0] == "type":
+                try:
+                    value = int(search[1])
+                except ValueError:
+                    value = 0
+                if value:
+                    domain.append(("event_type_id", "=", int(search[1])))
```

Comment:
If all you're using `searches` for is for filtering by type, it would be better if the parameter were called `type` directly and got an `int` or `List[int]`.